docs/testing-pre-releases.md

```diff
@@ -167,8 +154,8 @@ Run `cluster/get-kube-binaries.sh` to download the tarball with server binaries.
 
 > Inside release notes, usually there is a direct link for getting server binaries directly
 
-> `cluster/get-kube-binaries.sh` retrieves server binaries from `gs://kubernetes-release/release/{release}`
-  GCS bucket; you can use `gsutil` to get server binaries directly.
+> `cluster/get-kube-binaries.sh` retrieves server binaries from `https://dl.k8s.io/release/{release}`
```

Comment:
Were those fully depricated or were they moved somewhere else?

Reply:
i haven't checked. we should not ref /cluster tools from k/kubeadm, since /cluster should not be used directly by users. instead users should curl, perhaps.